tests/e2e/Output_Stream/infection.json.dist

```diff
@@ -0,0 +1,12 @@
+{
+    "timeout": 10,
+    "source": {
+        "directories": [
+            "src"
+        ]
+    },
+    "logs": {
+        "text": "php://stderr"
```

Comment:
Please, use `debug` logger as in other tests. It was specifically designed for e2e tests, and has more convenient output for tests.